src/main/resources/i18n/displayStrings.properties

```diff
@@ -35,9 +35,9 @@ shared.no=No
 shared.iUnderstand=I understand
 shared.na=N/A
 shared.shutDown=Shut down
-shared.reportBug=Report bug at Github issues
-shared.buyBitcoin=Buy bitcoin
-shared.sellBitcoin=Sell bitcoin
+shared.reportBug=Report bug at GitHub issues
+shared.buyBitcoin=Buy Bitcoin
+shared.sellBitcoin=Sell Bitcoin
```

Comment:
Some of the capitalizations of Bitcoin you've done in this PR are correct, and some should remain lower case. Please review the following resources regarding capitalization of the word bitcoin, and update the PR accordingly, thanks.

 - https://bitcoin.org/en/vocabulary#bitcoin
 - https://bitcoin.stackexchange.com/questions/20901/correct-use-of-the-word-bitcoin